localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```
@@ -18,3 +19,186 @@
+
+ innovation_kodo:0 "Kodo Taming"
+ innovation_kodo_desc:0 "Tauren legends abound regarding the great horned kodos of Kalimdor’s Barrens. Some myths tell of rare kodos that are bound to the spirits of the sky and storm, thus allowing the colossal beasts to harness the powers of lightning itself. While kodos are generally considered gentle by nature, they are fierce opponents when threatened and have occasionally been known to devour their enemies whole. Our tamers have successfully learned how to communicate and tame these grand creatures."
+ 
+ innovation_marauder:0 "Marauders"
+ innovation_marauder_desc:0 "Like most of thier kin, Marauders are savagery and brutality incarnate.  Marauders are fast, aggresive and show no mercy. Dawning the hides of their enemies Marauders are not for taking prisoners."
+    
+ innovation_magi:0 "Magis"
+ innovation_magi_desc:0 "Ogre Magis are skilled spellcasting warriors. The two-headed ogre spellcasters are typically larger and more intelligent than their one-headed brethren. They often enjoy an elevated position among the ogre clans"
+    
+ innovation_mountain_giant:0 "Mountain Giants"
+ innovation_mountain_giant_desc:0 "Mountain Giants are slow melee units that can take an incredible amount of damage. These creatures, created by Titans from solid rock ages ago, strive to protect the natural environment from anything that could harm it. Their titanic origins give them a naturally high resistance to physical and magical attacks."
+ 
+ innovation_naga_ground:0 "Naga Ground Tactics"
+ innovation_naga_ground_desc:0 "With our unique ground armaments, we have produced regiments like the Myrmidons and the Sirens."
+ 
+ innovation_naga_taming:0 "Naga Taming"
+ innovation_naga_taming_desc:0 "We have tamed the Couatl and enslaved the Sea Giants, and now we have encorporated them into our army."
+ 
+ innovation_savage:0 "Savages"
+ innovation_savage_desc:0 "Savages are fanatic and barbaric warriors of our clan, they are strong and stubborn, and show no mercy."
+ 
+ innovation_silithid:0 "Silithid"
+ innovation_silithid_desc:0 "Silithids are insectoid creatures that are very strong and sophisticated. The chitin plates of their bone armor are intertwined in such a way that it makes it incredibly hard. It seems that the Silithids' intention is to devour everything that stands in their way and that does not belong to their own ecology. This is the only thing they focus on, and this is why they pose a great threat to all life on Azeroth."
+
+ innovation_snow_hunter:0 "Snow Hunters"
+ innovation_snow_hunter_desc:0 "Our kind has had to adapt to the elements in order to survive.  The best of our warriors are the hunters who keep us safe."
+    
+ innovation_thornguard:0 "Thornguards"
+ innovation_thornguard_desc:0 "The Thornguards are the finest warriors from our tribe, who defend us from any outsiders.  They are heavily armed and armoredt They will stop any who dare to trespass or any who dare to not submit."
```

Another double space and minor typo fix:
```suggestion
 innovation_thornguard_desc:0 "The Thornguards are the finest warriors from our tribe, who defend us from any outsiders. They are heavily armed and armored They will stop any who dare to trespass or any who dare to not submit."
```